.github/workflows/integration.yaml

```diff
@@ -0,0 +1,67 @@
+name: Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
```

Comment:
I would define matrix strategy where we would define multiple java versions if needed, or it doesn't make sense in that case?

Reply:
We currently compile sql-runner targeting java 11 so don't think it matters which java version is installed as long as it's 11 or higher. However, once we support java 17 and compile the sql-runner targeting it, would we want to run it on java 11 to make sure it's backward compatible? That's of course as long as Java 11 is still supported for Flink. In that case, adding multiple java versions to run with would be useful.

Reply:
Let's keep the base line of java 17 for now.